google/cloud/storage/tools/run_testbench_utils.sh

```diff
@@ -105,5 +106,18 @@ start_testbench() {
     echo "Successfully connected to testbench [${TESTBENCH_PID}]"
   fi
 
+  port="${2:-0}"
+  local grpc_port=""
+  grpc_port=$(curl -s --retry 5 --retry-max-time 40 "http://localhost:${testbench_port}/start_grpc?port=${port}")
+
+  if [[ "$grpc_port" -eq "$grpc_port" ]]; then
```

Comment:
Not a review, more of a question. What does this self-comparison mean here?

Reply:
It's a [way](https://stackoverflow.com/a/19116862) to check if `grpc_port` is a valid integer or not. I added a commit to make the error message clearer and fixed an error where I mistook the difference between `[` and `[[`. PLease let me know if there is something that is not clear.

Reply:
Thanks, I learned something new.